setup.py

```diff
@@ -88,7 +88,7 @@ def build_extension(self, ext):
 
 setup(
   name = "RocketSim",
-  version = "1.2.0a11",
+  version = "1.3.0a1",
```

Comment:
Should probably be `1.2.0` to match the c++ lib version. Dropping the `a` since it's actually 1.2.0 released now.

Also update version to match in python/Makefile so GitHub action picks the right file